pythonchile_v2/templates/base.html

```diff
@@ -134,6 +134,14 @@
                         </span>
                     </a>
                     </li>
+		    <li class="list-inline-item">
+		    <a href="https://www.linkedin.com/groups/4929519/">
```

Comment:
```suggestion
		      <li class="list-inline-item">
		      <a href="https://www.linkedin.com/groups/4929519/">
```
para que quede con la indentación como los demás

Reply:
hago el cambio y vuelve a commitear?